pricegraph/src/num.rs

```diff
@@ -4,6 +4,10 @@ use primitive_types::U256;
 use std::cmp;
 use std::f64;
 
+/// The maximum rounding error, used for asserting that amounts and balances
+/// remain coherent for tests and when built in `debug` profile.
+pub const MAX_ROUNDING_ERROR: f64 = 0.1;
```

Comment:
Should this and MIN_AMOUNT be compile time constants or configurable?

Reply:
`MIN_AMOUNT` probably shouldn't as it is defined by the smart contract and set in stone at the moment.

For `MAX_ROUNDING_ERROR`, its only used for tests at the moment (since Wasm is built in `release`) and I think it should be selected based on the kind of calculations being done and isn;t really a configuration parameter.